contracts/interfaces/IVoucherKernel.sol

```diff
@@ -151,12 +151,6 @@ interface IVoucherKernel {
         address _newSeller
     ) external;
 
-    /**
-     * @notice Set the general cancelOrFault period, should be used sparingly as it has significant consequences. Here done simply for demo purposes.
-     * @param _cancelFaultPeriod   the new value for cancelOrFault period (in number of seconds)
-     */
-    function setCancelFaultPeriod(uint256 _cancelFaultPeriod) external;
```

Comment:
You should leave this in the interface. No problem if it's external here, but the actual implementation is public.

Reply:
Added back in.